source/operations/install-deploy-manage/deploy-operator-kustomize.rst

```diff
@@ -54,7 +54,7 @@ The following procedure uses ``kubectl -k`` to install the Operator from the Min
       :class: copyable
       :substitutions:
 
-      kubectl apply -k github.com/minio/operator\?ref=v|operator-version-stable|
+      kubectl apply -k "github.com/minio/operator\?ref=v|operator-version-stable|"
```

Comment:
I think you forgot to remove the escaping of the questionmark:
```suggestion
      kubectl apply -k "github.com/minio/operator?ref=v|operator-version-stable|"
```

Reply:
whoops!